mpyl-example.py

```diff
@@ -129,6 +129,11 @@ def main(log: Logger, args: argparse.Namespace):
         default=False,
         action="store_true",
     )
+    parser.add_argument(
+        "--stage",
+        help="Stage up to which to run. Prerequisite stages are also run, unless they're cached",
+        default=None,
```

Comment:
Didn't you have here a `Choice` so that we immediately check if the stage is correct?
EDIT: I see further down that you have it in the cli/build.py. Maybe good to use it here as well

Reply:
I'm actually not sure if we want to allow stage runs via the runner at all, what do you think?